tasks/scratch_build.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Tito release"
+  command: "tito release {{ releasers | join(' ') }} --scratch --arg source_dir=/tmp/pulp/{{ inventory_hostname }}"
```

Comment:
Why do you specify a source_dir argument here?

Reply:
It's probably not expressed well enough, but this scratch task works with the 'nightly' workflow, so it still generates the tarball from the upstream github source.  As support is added in for other releasers, I plan on improving the scratch_build.yaml to support them as well.

Reply:
Gotcha.